concourse/tasks/update-ecs-service.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env sh
+
+set -eu
+
+mkdir -p ~/.aws
+
+cat <<EOF > ~/.aws/config
+[profile default]
+role_arn = $ASSUME_ROLE_ARN
+credential_source = Ec2InstanceMetadata
+EOF
+
+new_task_definition_arn="$(cat "task-definition-arn/task-definition-arn")"
+if [ -z "${new_task_definition_arn}" ]; then
+  echo "failed to retrieve new task definition, exiting..."
```

Comment:
might be good to add the ECS service name as you did for the log messages below, i.e.
`failed to retrieve a new task definition for $ECS_SERVICE service, exiting...`

Reply:
26f34f3